exporter/trace/stackdriver/stackdriver.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2019, OpenTelemetry Authors
+// Copyright 2017, OpenCensus Authors
```

Comment:
multiple copyright statements. OR is it deliberately added to acknowledge that it is based from opencensus stackdriver exporter?

Reply:
I meant to put multiple copyright owners because most of codes are from OpenCensus Stackdriver Exporter.
https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/master/stackdriver.go

Reply:
I think putting just one copyright with OpenTelemetry Authors is fine. We will have a reference to OpenCensus in main License file.

Reply:
Removed copyright statements of OpenCensus from license headers.